app/src/main/java/org/p2p/wallet/home/events/StrigaBannersLoader.kt

```diff
@@ -0,0 +1,31 @@
+package org.p2p.wallet.home.events
+
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.launch
+import org.p2p.core.common.di.AppScope
+import org.p2p.wallet.common.feature_toggles.toggles.remote.StrigaSignupEnabledFeatureToggle
+import org.p2p.wallet.home.ui.main.HomeInteractor
+import org.p2p.wallet.striga.user.interactor.StrigaUserInteractor
+
+class StrigaBannersLoader(
+    private val strigaUserInteractor: StrigaUserInteractor,
+    private val strigaSignupEnabledFeatureToggle: StrigaSignupEnabledFeatureToggle,
+    private val appScope: AppScope,
+    private val homeInteractor: HomeInteractor
+) : HomeScreenLoader {
+
+    override suspend fun onLoad() {
+        if (!strigaSignupEnabledFeatureToggle.isFeatureEnabled) {
```

Comment:
I suggest to consider some mechanism that would check whether a Loader should be enabled or disabled by getting some boolean value from an implementation
For example you could add function to the HomeScreenLoader:
```
interface HomeScreenLoader {
    // ... 
    fun isEnabled(): Boolean = true
}
```

and here in StrigaBannersLoader:
```
class StrigaBannersLoader {
  // ... 
  override fun isEnabled() = strigaSignupEnabledFeatureToggle.isFeatureEnabled
}
```

Thus we would be able to switch them dynamically dependning on feature toggle or any other stuff. 
Btw, `fun isEnabled()` could be even a suspend function to check a state asynchronously